src/test/java/com/example/universe/simulator/entityservice/integration/AbstractIntegrationTest.java

```diff
@@ -24,16 +27,25 @@ abstract class AbstractIntegrationTest extends AbstractMockMvcTest {
 
     private static final RabbitMQContainer RABBITMQ_CONTAINER;
     private static final PostgreSQLContainer<?> POSTGRESQL_CONTAINER;
+    private static final GenericContainer<?> REDIS_CONTAINER;
 
     @Autowired
     private ApplicationEvents applicationEvents;
 
+    @Autowired
+    protected ModelMapper modelMapper;
```

Comment:
There's already a `modelMapper` field in `AbstractMockMvcTest`. `AbstractIntegrationTest` is a child of `AbstractMockMvcTest`, so you it will inherit this field.

Action points:

1. delete the `modelMapper` field in `AbstractMockMvcTest`

Reply:
Removed.